scenario/SupplyScenario.ts

```diff
@@ -250,7 +250,7 @@ scenario(
         asset: baseAsset.address,
         amount: 100n * scale,
       })
-    ).to.be.revertedWith('ERC20: transfer amount exceeds allowance');
+    ).to.be.reverted;
```

Comment:
`WETH` reverts without reason strings, unfortunately ☹️

Reply:
This kinda sucks, another case where I think we should at least preserve the error information in a comment in case we find a way to restore strength later

Reply:
sure, added the error messages back as comments.

as long as we're running these scenarios against a WETH-based deployment we'll have to stick with `.reverted`, though, since the WETH contract we're using isn't ever going to return revert strings.

Reply:
maybe one day we'll have network specific revert reasons 🤷

Reply:
oh, that's an interesting idea.

we could write a wrapper function that the function to execute and a map of networks to the expected error message. for anything